stdlib/3/multiprocessing/context.pyi

```diff
@@ -0,0 +1,113 @@
+# Stubs for multiprocessing.context
+
+# NOTE: These are incomplete!
+
+from logging import Logger
+from typing import Any, Callable, Optional, List, Sequence, Tuple, Union
+
+class BaseContext(object):
+    def cpu_count(self) -> int: ...
+
+    # TODO: change return to SyncManager once a stub exists in multiprocessing.managers
+    def Manager(self) -> Any: ...
+
+    # TODO: change return to Pipe once a stub exists in multiprocessing.connection
+    def Pipe(self, duplex: bool) -> Any: ...
+
+    # TODO: change return to Lock once a stub exists in multiprocessing.synchronize
+    def Lock(self) -> Any: ...
+
+    # TODO: change return to RLock once a stub exists in multiprocessing.synchronize
+    def RLock(self) -> Any: ...
+
+    # TODO: change lock param to Optional[Union[Lock, RLock]] when stubs exists in multiprocessing.synchronize
+    # TODO: change return to Condition once a stub exists in multiprocessing.synchronize
+    def Condition(self, lock: Optional[Any] = None) -> Any: ...
+
+    # TODO: change return to Semaphore once a stub exists in multiprocessing.synchronize
+    def Semaphore(self, value: Optional[int] = 1) -> Any: ...
+
+    # TODO: change return to BoundedSemaphore once a stub exists in multiprocessing.synchronize
+    def BoundedSemaphore(self, value: Optional[int] = 1) -> Any: ...
+
+    # TODO: change return to Event once a stub exists in multiprocessing.synchronize
+    def Event(self) -> Any: ...
+
+    # TODO: change return to Barrier once a stub exists in multiprocessing.synchronize
+    def Barrier(self, parties: int, action: Optional[Callable] = None, timeout: Optional[int] = None) -> Any: ...
+
+    # TODO: change return to Queue once a stub exists in multiprocessing.queues
+    def Queue(self, maxsize: Optional[int] = 0) -> Any: ...
```

Comment:
I don't think maxsize is Optional (Optional means that it can be None, not that the argument is optional). queue.py seems to assume it's an int.